includes/class.llms.install.php

```diff
@@ -487,6 +516,14 @@ private static function get_schema() {
   KEY actor_id (`actor_id`),
   KEY object_id (`object_id`)
 ) $collate;
+CREATE TABLE `{$wpdb->prefix}lifterlms_sessions` (
+  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+  `user_id` char(32) NOT NULL,
```

Comment:
I don't like it either... I wanted it to be "key" but that's a reserved word (I guess?) and was causing issues when running tests (although it worked otherwise).

I changed it last minute before pushing. Maybe using `session_key` would be more appropriate, I just feel like it's saying the "session session_key", or, as it would be written in a join query `session.session_key`. That redundancy bothers me to no end but here maybe it's better than having a semantically incorrect column name.

Reply:
I wouldn't like `session.session_key` either but you know what can we do?! So thanks. :)